mods/classic-taskbar-buttons-lite.wh.cpp

```diff
@@ -0,0 +1,105 @@
+// ==WindhawkMod==
+// @id              classic-taskbar-buttons-lite
+// @name            Classic Taskbar 3D buttons Lite
+// @description     Lightweight mod, restoring the 3D buttons in classic theme
+// @version         1.2
+// @author          Anixx
+// @github          https://github.com/Anixx
+// @include         explorer.exe
+// @compilerOptions -lgdi32
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Classic Taskbar 3D buttons
+Lightweight mod which restores 3D buttons on taskbar when using Windows Classic theme. 
+The idea is based on the mod by Aubymori (https://github.com/aubymori).
+
+Before:
+
+![Before](https://i.imgur.com/jupSjfl.png)
+
+After:
+
+![After](https://i.imgur.com/Jz4EkRQ.png)
+
+*/
+// ==/WindhawkModReadme==
+
+#include <windhawk_utils.h>
+
+
+typedef struct tagBUTTONRENDERINFOSTATES {
+    char data[12];
+} BUTTONRENDERINFOSTATES, *PBUTTONRENDERINFOSTATES;
+
+/* Draw taskbar item */
+typedef void (* CTaskBtnGroup__DrawBar_t)(void *, HDC, void *, void *);
```

Comment:
Calling conventions are missing, which is likely to cause a crash for 32-bit targets. Please add calling conventions or, if the mod doesn't support 32-bit, limit it to 64-bit, which will also make compilation faster. See also:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions